classification/audio_analysis.py

```diff
@@ -0,0 +1,89 @@
+import os
+import sys
+import re
+import csv
+import subprocess
+
+# Start of importing the utilities module
+sys.path.insert(0, os.path.abspath("../utilities/"))
+import utilities.get_song_id as s_id
+# End of importing the utilities module
+
+from classification.extractor.low_level_data_extractor import make_low_level_data_file
+from classification.classifier.profile_data_extractor import get_classifier_data
+from similarity.split_song import split_song
+from utilities.filehandler.handle_audio import get_MonoLoaded_Song
+from utilities.filehandler.handle_path import get_absolute_path
+from essentia.standard import MonoWriter
+
+from multiprocessing import Pool
```

Comment:
Imports should be done in a "standard order" according to [pep8](https://pep8.org/#imports):

> Imports should be grouped in the following order:
> standard library imports
> related third party imports
> local application/library specific imports